packages/instant-meilisearch/src/adapter/search-request-adapter/__tests__/geo-rules.test.ts

```diff
@@ -1,82 +1,82 @@
-import { adaptGeoPointsRules } from '../geo-rules-adapter'
+import { adaptGeoFilter } from '../geo-rules-adapter'
 
-test('Adapt geoPoints rules without a boundingBox', () => {
-  const rules = adaptGeoPointsRules()
-  expect(rules).toBeUndefined()
+test('Adapt geoPoints filter without a boundingBox', () => {
```

Comment:
`geoPoints filter` or `geo filter` (as in `adaptGeoFilter`)? I'm confused now :D

Reply:
I agree, let me rename all the tests